add table of bold parameters (#2180)
OffchainLabs/arbitrum-docs: arbitrum-docs/launch-orbit-chain/02-configure-your-chain/advanced-configurations/04-bold.mdx

---
title: 'BoLD configuration parameters'
sidebar_label: 'BoLD configuration parameters'
description: 'Learn about the various parameters that can be configured for Arbitrum BoLD'
author: dlee
sme: dlee
user_story: As a current or prospective Orbit chain owner, I need to understand the various configuration parameters with Arbitrum BoLD
content_type: how-to
---

<a data-quicklook-from="bold">Arbitrum BoLD</a> is the latest iteration of the Optimistic Rollup dispute
protocol used by Arbitrum chains today, like Arbitrum One. This page is intended for Orbit chain owners/operators
and assumes knowledge of Arbitrum BoLD is and experience upgrading an Arbitrum chain's Rollup contracts.

> To read more about Arbitrum BoLD, please refer to the [Gentle Introduction for BoLD](../../../how-arbitrum-works/bold/gentle-introduction.mdx). To learn more about some guidelines/best-practices for using BoLD on your Orbit chain, please refer to our guide on [BoLD adoption for Orbit chains](../../bold-adoption-for-orbit-chains.mdx).

Reviewer: We should probably include a link here to the Orbit update guide once it's live.

Author: I'll do that !

:::caution
As mentioned in our guide on [BoLD adoption for Orbit chains](../../bold-adoption-for-orbit-chains.mdx), the recommendation is that Orbit chains keep validation permissioned by having `disableValidatorWhitelist` be `false` (which is the default) and by having a list of validators on the whitelist via the `validators[]` array.
:::

## Table of Arbitrum BoLD parameters

> The following configuration parameters can be applied when deploying or managing your Orbit chain. For an example of a configuration, feel free to reference [Arbitrum One's configuration](https://github.com/OffchainLabs/nitro-contracts/blob/main/scripts/files/configs/arb1.ts).

Reviewer: Arbitrum One has the validator whitelist disabled, and I'd be concerned that users just copy/paste and change a few parameters. We can maybe reference Nova's configuration, include here an example object, or point at the section in the page I created that includes the example object.
What do you think?

> The following configuration parameters can be applied when deploying or managing your Orbit chain. For an example of a configuration, feel free to reference [Arbitrum One's configuration](https://github.com/OffchainLabs/nitro-contracts/blob/main/scripts/files/configs/arb1.ts).

Reviewer: Arbitrum One has the validator whitelist disabled, and I'd be concerned that users just copy/paste and change a few parameters. We can maybe reference Nova's configuration, include here an example object, or point at the section in the page I created that includes the example object.
What do you think?

Reviewer: Yeah, the `config.ts.example` you linked is for creating rollups, instead of enabling BoLD. BoLD configurations are in this folder: https://github.com/OffchainLabs/nitro-contracts/blob/main/scripts/files/configs/nova.ts

There's one for Arb1, Nova, Sepolia and Local. But none is good enough for our purposes 😅 .

I think it's ok to just include a raw config object here like done in [this section](https://arbitrum-docs-git-orbit-upgrade-bold-offchain-labs.vercel.app/launch-orbit-chain/maintain-your-chain/upgrade-to-bold#step-2-configure-your-chain-parameters).

---
title: 'BoLD configuration parameters'
sidebar_label: 'BoLD configuration parameters'
description: 'Learn about the various parameters that can be configured for Arbitrum BoLD'
author: dlee
sme: dlee
user_story: As a current or prospective Orbit chain owner, I need to understand the various configuration parameters with Arbitrum BoLD
content_type: how-to
---

<a data-quicklook-from="bold">Arbitrum BoLD</a> is the latest iteration of the optimistic rollup dispute

> protocol used by Arbitrum chains today, like Arbitrum One. This page is meant for Orbit chain owners/operators

Reviewer 2: ```suggestion
Protocol used by Arbitrum chains today, like Arbitrum One. This page is intended for Orbit chain owners/operators
```

and assumes knowledge of Arbitrum BoLD is and experience upgrading

> an Arbitrum chain's Rollup contracts.{' '}

Reviewer 2: hanging squiggly brackets

> an Arbitrum chain's rollup contracts.{' '}

Reviewer 2: the hanging squiggly brackets?

> To read more about what Arbitrum BoLD is, please refer to the [Gentle Introduction for BoLD](../../../how-arbitrum-works/bold/gentle-introduction.mdx). To learn more about some guidelines/best-practices for using BoLD on your Orbit chain, please refer to our guide on [BoLD Adoption for Orbit chains](../../bold-adoption-for-orbit-chains.mdx).

Reviewer 2: ```suggestion
To read more about Arbitrum BoLD, please refer to the [Gentle Introduction for BoLD](../../../how-arbitrum-works/bold/gentle-introduction.mdx). To learn more about some guidelines/best-practices for using BoLD on your Orbit chain, please refer to our guide on [BoLD adoption for Orbit chains](../../bold-adoption-for-orbit-chains.mdx).
```

:::caution

> As mentioned in our guide on [BoLD Adoption for Orbit chains](../../bold-adoption-for-orbit-chains.mdx), it is highly recommended that Orbit chains keep validation permissione by having `disableValidatorWhitelist` be `false` (which is the default) and by having a list of validators on the whitelist via the `validators[]` array.

Reviewer 2: ```suggestion
As mentioned in our guide on [BoLD adoption for Orbit chains](../../bold-adoption-for-orbit-chains.mdx), the recommendation is that Orbit chains keep validation permissioned by having `disableValidatorWhitelist` be `false` (which is the default) and by having a list of validators on the whitelist via the `validators[]` array.
```

:::

## Table of Arbitrum BoLD parameters

> The following configuration parameters can be used when deploying or managing your Orbit chain. For an example of a configuration, feel free to reference [Arbitrum One's configuration](https://github.com/OffchainLabs/nitro-contracts/blob/main/scripts/files/configs/arb1.ts).

Reviewer 2: ```suggestion
The following configuration parameters can be applied when deploying or managing your Orbit chain. For an example of a configuration, feel free to reference [Arbitrum One's configuration](https://github.com/OffchainLabs/nitro-contracts/blob/main/scripts/files/configs/arb1.ts).
```
